### PR TITLE
Problem: #69 App architecture inefficiencies and overhead

### PR DESCRIPTION
**Pull request by janat08**
Solution: Moved most of lib into imports. #69

### CODE REVIEW

**Comment by gazhayes** [comment]
Can you resolve the conflicts

**Comment by janat08** [comment]
how to tell what the conflict is about?

File: imports/api/indexDB.js [hunk 1]
```diff
@@ -7,43 +7,38 @@ import { HashUnits } from './hashing/hashUnits.js'
...
 
 import { Bounties } from './bounties/bounties.js'
 import { BountyTypes } from './bounties/bountyTypes.js'
 import { REWARDCOEFFICIENT } from './bounties/REWARDCOEFFICIENT.js'
 
-///////////////////////////////////////////////////////////
+import { WalletImages } from './wallet/walletImages.js'
+import { Wallet } from './wallet/wallet.js'
 
-import { Currencies } from '/imports/api/coins/Currencies.js'
```
Comment (gazhayes) [inline imports/api/indexDB.js]: Capital

File: imports/api/indexDB.js [hunk 2]
```diff
@@ -7,43 +7,38 @@ import { HashUnits } from './hashing/hashUnits.js'
...
-import { RatingsTemplates } from '/lib/database/Ratings'
-import { EloRankings } from '/lib/database/Ratings'
+import { Features } from './features/features.js'
 
-import { WalletImages } from '/lib/database/Images'
-import { Wallet } from '/lib/database/Wallet'
+import { Communities } from './communities/communities.js'
 
-import { Codebase } from '/lib/database/Codebase'
+import { Currencies } from '/imports/api/coins/currencies.js'
```
Comment (gazhayes) [inline imports/api/indexDB.js]: Lower case

**Pull request closed without merging**